cmd/sync/aws.go

```diff
@@ -167,7 +167,7 @@ func (client *AWSClient) GetPrivateIPsForScalingGroup(name string) ([]string, er
 
 type awsConfig struct {
 	Region    string
-	Upstreams []awsUpstream
+	Upstreams []*awsUpstream
```

Comment:
I wonder if this is still necessary considering the change in `GetUpstreams()`method? If we revert it, the unit test will still pass.

Reply:
I'll check thanks

Reply:
Yup, changed.